src/Sds011.cpp

```diff
@@ -138,9 +138,16 @@ bool Sds011::crc_ok() {
     for (int i = 2; i < 8; ++i) {
         crc += _buf[i];
     }
+    if (crc != _buf[8]) checksum_errors++;
     return crc == _buf[8];
 }
 
+bool Sds011::crc_stats(uint32_t& err, uint32_t& total){
+    err = checksum_errors;
+    total = processed_responses;
+    return true;
+};
+
```

Comment:
References to ints, I don't like that, you could either return a struct, or just provide two accessors.
Also, I would like to be able to reset the counters.

Reply:
I was copying style from other functions like `query_data_auto`